docs/how-to-guides/monitoring-with-prometheus-operator.md

```diff
@@ -0,0 +1,139 @@
+
+<!---
+TODO: Once we have a tutorial on how to deploy and configure cert-manager, then point the following to that tutorial.
+-->
+
+* [cert-manager](https://cert-manager.io/docs/) deployed on the cluster if you wish to [expose Grafana using Ingress](#using-ingress).
+
+
+## Deploy Prometheus Operator
+
+### Configure Prometheus Operator
+
+Create a file named `monitoring.lokocfg` with the contents below:
+
+```tf
+component "prometheus-operator" {}
+```
+
+There are many optional parameters one can provide for prometheus-operator to tweak its behaviour. Find information about all the parameters in the prometheus-operator [config reference doc](../configuration-reference/components/prometheus-operator.md).
+
+### Install Prometheus Operator
+
+Execute the following command to install prometheus-operator:
+
+```bash
+lokoctl component apply prometheus-operator
+```
+
+Within a few minutes, the prometheus-operator pods will be in `Running` state in `monitoring` namespace. Verify it by executing the following command:
+
+```bash
+kubectl -n monitoring get pods
+```
+
+## Accessing Prometheus Operator sub-components
+
+### Accessing Prometheus
+
+Execute the following command to forward port `9090` locally to the Prometheus pod:
+
+```bash
+kubectl -n monitoring port-forward svc/prometheus-operator-prometheus 9090
+```
+
+Open the following URL: [http://localhost:9090](http://localhost:9090).
+
+### Accessing Alertmanager
+
+Execute the following command to forward port `9093` locally to the Alertmanager pod:
+
+```bash
+kubectl -n monitoring port-forward svc/prometheus-operator-alertmanager 9093
+```
+
+Open the following URL: [http://localhost:9093](http://localhost:9093).
+
+### Accessing Grafana
+
+#### Using port forward
+
+Execute the following command to forward port `8080` locally to the Grafana dashboard pod on port `80`:
+
+```bash
+kubectl -n monitoring port-forward svc/prometheus-operator-grafana 8080:80
+```
+
+Obtain the password for the `admin` Grafana user by running the following command:
+
+```bash
+kubectl -n monitoring get secret prometheus-operator-grafana -o jsonpath='{.data.admin-password}' | base64 -d && echo
+```
+
+Open the following URL: [http://localhost:8080](http://localhost:8080). Enter the username `admin` and password obtained from the previous step.
+
+#### Using Ingress
+
+To expose grafana to the internet using Ingress, provide the host field. The configuration for grafana in prometheus-operator should look like the following:
+
+```tf
+component "prometheus-operator" {
+  grafana {
+    ingress {
+      host = "grafana.<cluster name>.<DNS zone>"
+    }
+  }
+}
+```
+
+**NOTE**: On Packet make a DNS entry for `grafana.<cluster name>.<DNS zone>` against the Packet EIP as mentioned in the
+["Setting up an HTTP ingress controller on Lokomotive with MetalLB and Contour on Packet"](./ingress-with-contour-metallb.md) guide.
```

Comment:
```suggestion
>NOTE: On Packet, create a DNS entry for `grafana.<cluster name>.<DNS zone>` and point it to the Packet EIP as mentioned [here](./ingress-with-contour-metallb.md).
```

Reply:
Again, I feel like we should fix the issue which makes Packet a "special case". I realize this is out of scope though :shrug: